easybuild/easyconfigs/i/IOR/IOR-3.2.1-foss-2019b.eb

```diff
@@ -0,0 +1,30 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'ConfigureMake'
+
+name = "IOR"
+version = "3.2.1"
+
+homepage = 'https://github.com/IOR-LANL/ior'
+description = """ The IOR software is used for benchmarking parallel file systems using POSIX, MPIIO,
+ or HDF5 interfaces. """
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+source_urls = [('https://github.com/hpc/ior/archive')]
+sources = ['%(version)s.tar.gz']
+checksums = ['ebcf2495aecb357370a91a2d5852cfd83bba72765e586bcfaf15fb79ca46d00e']
+
+dependency = [('Autotools', '20180311')]
```

Comment:
optional deps like HDF5?

Reply:
Prefer not to really but could do that.

Reply:
I just thought i'd mention it because the description text specifically mentions the HDF5 part.

Reply:
doesn't work with hdf5 1.10, dropping it again